src/components/application_manager/test/resumption/resumption_data_db_test.cc

```diff
@@ -134,6 +128,14 @@ class ResumptionDataDBTest : public ResumptionDataTest {
 
   // Check that db includes tables with given elements
   void CheckSavedDB();
+
+  // TODO(EKuliiev@luxoft.com): if 'is_in_file' flag set to 'true'
+  // TestResumptionDataDB fails while instantiation at
+  // 'sdl_core/src/components/application_manager/src/resumption/resumption_data_db.cc:59
```

Comment:
@EKuliiev line number can be changed at any moment. Therefore its specifying is meaningless

Reply:
agree. line number removed.